pointerevents/compat/pointerevent_touch-action_two-finger_interaction-manual.html

```diff
@@ -0,0 +1,100 @@
+    <link rel="stylesheet" type="text/css" href="../pointerevent_styles.css">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script type="text/javascript" src="../pointerevent_support.js"></script>
+    <script type="text/javascript">
+      var detected_pointertypes = {};
+      var event_log = [];
+      var active_pointers = 0;
+
+      function resetTestState() {
+        detected_eventTypes = {};
+        event_log = [];
+        active_pointers = 0;
+      }
+
+      function run() {
+        var test_pointer_event = setup_pointerevent_test("touch-action test for two-finger interaction", ["touch"]);
+
+        on_event(document.getElementById("done"), "click", function() {
+          test_pointer_event.step(function () {
+            assert_equals(active_pointers, 0);
+
+            var expected_events =
+              "pointerdown@black, pointerdown@black, pointercancel@black, pointercancel@black, " +
+              "pointerdown@grey, pointerdown@grey, pointerup@grey, pointerup@grey";
+            assert_equals(event_log.join(", "), expected_events);
+          });
+          test_pointer_event.done();
+        });
+
+        var targets = [document.getElementById("black"), document.getElementById("grey")];
+
+        ["pointerdown", "pointerup", "pointercancel"].forEach(function(eventName) {
+          targets.forEach(function(target){
+            on_event(target, eventName, function (event) {
+              detected_pointertypes[event.pointerType] = true;
+              event_log.push(event.type + "@" + event.target.id);
+
+              if (event.type == "pointerdown") {
+                active_pointers++;
+                event.target.setPointerCapture(event.pointerId);
+              } else {
+                active_pointers--;
+              }
+            });
+          });
+        });
+      }
+    </script>
+    <style>
+      .box {
+        width: 200px;
+        height: 150px;
+        float: left;
+        margin: 10px;
+      }
+
+      #black {
+        touch-action: pan-x pan-y;
+        background-color: black;
+      }
+
+      #grey {
+        touch-action: pinch-zoom;
+        background-color: grey;
+      }
+
+      #done {
+        background-color: green;
+      }
+    </style>
+  </head>
+  <body onload="run()">
+    <h1>Pointer Event: touch-action test for two-finger interaction<h1>
+    <h2 id="pointerTypeDescription"></h2>
+    <h4>
+      Tests that a two-finger pan gesture is cancelled in 'touch-action: pan-x pan-y' but is allowed in 'touch-action: pinch-zoom'
+    </h4>
+    <ol>
+      <li>Touch on Black with two fingers and drag both fingers down at same speed.</li>
+      <li>Touch on Grey with two fingers and drag both fingers down at same speed.</li>
+      <li>Tap on Green.</li>
+    </ol>
+    <div class="box" id="black"></div>
+    <div class="box" id="grey"></div>
+    <div class="box" id="done"></div>
```

Comment:
Would it be better to just have a complete button instead of a box for the finish action?